styles/atom-ide-outline.less

```diff
@@ -16,10 +16,11 @@
 }
 
 .outline-view {
+  margin-left: 10px; // overal left margin
```

Comment:
This creates a 10px gap between item highlighting and pane border. IMO this is ugly. I would suggest avoiding positive margins.

Reply:
It can be replaced with `padding-left: 5px;` under `.ul`

Reply:
That won't help though. We need `span` items to be flush on left side, because those are the ones that are highlighted.

Gap:
![image](https://user-images.githubusercontent.com/7275622/111147120-34f79d00-859b-11eb-9295-b8db21b6e35f.png)
![image](https://user-images.githubusercontent.com/7275622/111147819-f1516300-859b-11eb-8171-0791abcf7698.png)

Just increase `@padding-side`. Lose the margin.

No gap:
![image](https://user-images.githubusercontent.com/7275622/111147284-5bb5d380-859b-11eb-80ea-908f389266ae.png)

Reply:
Thanks. Fixed in 
https://github.com/atom-community/atom-ide-outline/pull/96/commits/019f508f87a4e4e4b8a273cb6963436565a57cf3